clients/pubsub-common/src/main/java/org/eclipse/hono/client/pubsub/PubSubConfigProperties.java

```diff
@@ -0,0 +1,42 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package org.eclipse.hono.client.pubsub;
+
+/**
+ * Common configuration properties required for access to Pub/Sub.
+ */
+public class PubSubConfigProperties {
+
+    private String projectId = null;
+
+    /**
+     * Creates properties based on existing options.
+     *
+     * @param options The options to copy.
+     */
+    public PubSubConfigProperties(final PubSubPublisherOptions options) {
+        setProjectId(options.projectId().orElse(null));
+    }
+
+    public String getProjectId() {
```

Comment:
can you please add (meaningful) JavaDoc explaining the semantics of the property?

Reply:
Sure, JavaDoc is added now